projects/biogears/libBiogears/src/cdm/patient/assessments/SECompleteBloodCount.cpp

```diff
@@ -103,6 +105,8 @@ void SECompleteBloodCount::Unload(CDM::CompleteBloodCountData& data)
     data.NeutrophilCellCount(std::unique_ptr<CDM::ScalarAmountPerVolumeData>(m_NeutrophilCellCount->Unload()));
   if (HasRedBloodCellCount())
     data.RedBloodCellCount(std::unique_ptr<CDM::ScalarAmountPerVolumeData>(m_RedBloodCellCount->Unload()));
+  if (HasNeutrophilCount())
```

Comment:
I think this is an error and you ment to add the LymphocyteCellCount unload/load logic